dev-docs/modules/floors.md

```diff
@@ -474,6 +481,77 @@ Additionally skipRate is supplied at model group level where model1 will skip fl
 }
 {% endhighlight %}
 
+## Custom Schema Fields
+
+Custom schema fields are fields the Floors Module does support out of the box. To use a custom schema field, one needs to perform twp steps:
```

Comment:
does 'not'

Reply:
of course they do not support them. Thanks for the catch.